From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nick Chan <towinchenmi@gmail.com>
Date: Thu, 7 Aug 2025 23:52:39 +0800
Subject: [PATCH 04/21] dt-bindings: mfd: apple,smc: Add Apple A11 and T2

The communication format of the SMC in Apple A11 and T2 are compatible
with the existing drivers so add their compatibles.

Signed-off-by: Nick Chan <towinchenmi@gmail.com>
---
 Documentation/devicetree/bindings/mfd/apple,smc.yaml | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/mfd/apple,smc.yaml b/Documentation/devicetree/bindings/mfd/apple,smc.yaml
index 34ce04861..16702250a 100644
--- a/Documentation/devicetree/bindings/mfd/apple,smc.yaml
+++ b/Documentation/devicetree/bindings/mfd/apple,smc.yaml
@@ -17,7 +17,10 @@ properties:
   compatible:
     oneOf:
       - items:
-          - const: apple,t6020-smc
+          - enum:
+              - apple,t6020-smc
+              - apple,t8012-smc
+              - apple,t8015-smc
           - const: apple,t8103-smc
       - items:
           - enum:
@@ -40,6 +43,10 @@ properties:
   mboxes:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+    description: SMC CPU power domain
+
   gpio:
     $ref: /schemas/gpio/apple,smc-gpio.yaml
 
